### ISSUE TITLE
Training/Testing on MPI-INF-3DHP

### ISSUE THREAD

**Issue by alexj94** [QitaoZhao/ContextAware-PoseFormer#7]
Hi @QitaoZhao,

thanks for your very interesting work!

Could you please clarify the following points:
- did you train on the MPI-INF-3DHP training set and then evaluate on the test set?
OR
- did you directly evaluate your model pre-trained on Human3.6M on the MPI-INF-3DHP test set (cross-dataset scenario)?

Thanks in advance for your response.

**Comment by QitaoZhao** [comment]
We did not do cross-dataset evaluation (we did the first option).

**Comment by alexj94** [comment]
> We did not do cross-dataset evaluation (we did the first option).

Thanks for your quick answer. 

Did you follow a specific protocol for training? I'm referring especially to the crop of the image you give as input to the 2D pose estimation (HRNet). Since there is no bounding box annotations in MPI-INF-3DHP, how did you crop the image?

**Comment by QitaoZhao** [comment]
You can look at https://github.com/QitaoZhao/ContextAware-PoseFormer/blob/a2456578e8cd25f9fd99dacdf81d2e3623ca127b/H36M-Toolbox/generate_labels_h36m.py#L162 where we infer the bounding box from 3D joints. (we use the script for both datasets)

**Comment by alexj94** [comment]
Ok perfect thanks a lot!
